projects/js-packages/components/components/admin-section/style.scss

```diff
@@ -0,0 +1,18 @@
+@import '~@automattic/jetpack-base-styles/style';
+
+// This should go to base styles?
```

Comment:
That's a good question. Any details why we're applying these styles?

Reply:
I believe changes to `#wpcontent` and `#wpwrap` styling should not go into any RNA packages.
I'd rather we wouldn't edit them at all, but if we do, we should put that in the Composer My Jetpack package, not in the RNA.
Otherwise somebody will get their whole WP-admin styling modified by simply importing an RNA component. I don't think we should allow that to happen.

Reply:
@leogermani can you explain more for me what these changes do at wp structure? (just to know more about it).

Reply:
Ok. Saw your comment here: https://github.com/Automattic/jetpack/pull/21826#issuecomment-981912803